assets/javascript/app.js

```diff
@@ -24,4 +24,20 @@ function backToTopButtonOffset() {
 function scrollBackToTopFunction() {
   document.body.scrollTop = 0;
   document.documentElement.scrollTop = 0;
-}
\ No newline at end of file
+}
+
+function searchFunction() {
+  var input, filter, cards, cardContainer, h5, title, i;
+  input = document.getElementById("myFilter");
+  filter = input.value.toUpperCase();
+  cardContainer = document.getElementById("myItems");
```

Comment:
@Forchapeatl 
It seems that you've get the element by `id`, but didn't define the `id` in the document.

Reply:
Thank you very much @ShehrozIrfan